tests/integration/pybaseball/datahelpers/test_caching.py

```diff
@@ -0,0 +1,30 @@
+from unittest.mock import patch
```

Comment:
This entire file should be gone in the final version I think

Reply:
Vehemently and completely disagree. This ensures that the cache does what it is intended to do, and that any future modifications can be done safely and without regressions. The code code coverage on the cache has 100% coverage, so every line is under test. Why would we want to NOT test code?

Reply:
Minimize the CacheType thingy?

Reply:
Yeah that's definitely going to happen, just waiting for everyone to get their feedback in so that we can make sure we have the correct, minimal cache options.